src/orga/guidelines.rst

```diff
@@ -100,8 +100,7 @@ See `conda-forge.github.io#1070 <https://github.com/conda-forge/conda-forge.gith
 Fixing Broken Packages
 ======================
 
-Sometimes you need to remove a package from the conda-forge channel on Anaconda.org. The reasons
-for this are many, but the ones that immediately come to mind are:
+Sometimes, you need to remove a package from the conda-forge channel on Anaconda.org. There can be many reasons for this, but the ones that immediately come to mind are:
```

Comment:
Sometimes, we put some spaces between those lines to help reviewing it (as we are able to directly point to the line we want to comment or modify)